packages/react/dropdown-menu/src/DropdownMenu.tsx

```diff
@@ -164,20 +164,21 @@ const DropdownMenuTrigger = React.forwardRef<DropdownMenuTriggerElement, Dropdow
           aria-expanded={context.open ? true : undefined}
           aria-controls={context.open ? context.contentId : undefined}
           data-state={context.open ? 'open' : 'closed'}
+          disabled={disabled}
```

Comment:
Should we add the `data-disabled` attr as well?

Reply:
Good shout, let's add it for consistency.